src/test/spec/mod.rs

```diff
@@ -97,10 +97,8 @@ where
     G: Future<Output = ()>,
     T: DeserializeOwned,
 {
-    let json: Value = serde_json::from_reader(File::open(path.as_path()).unwrap()).unwrap();
-
-    // Printing the name of the test file makes it easier to debug deserialization errors.
-    println!("Running tests from {}", path.display());
+    let json: Value = serde_json::from_reader(File::open(path.as_path()).unwrap())
```

Comment:
since we print the path when deserialization fails, I removed the extra print here since it clashed with the new logging changes I made in the runner itself.